builtin/providers/google/resource_dataflow_job.go

```diff
@@ -0,0 +1,200 @@
+package google
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/dataflow/v1b3"
+)
+
+func resourceDataflowJob() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDataflowJobCreate,
+		Read:   resourceDataflowJobRead,
+		Delete: resourceDataflowJobDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"gcs_path": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"temp_location": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"zone": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"max_workers": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  1,
```

Comment:
Good question... will the value just be null? I sort of assumed that wasn't allowed.